docs/api/token-api.mdx

```diff
@@ -0,0 +1,227 @@
+---
+title: (Alpha) Token API
```

Comment:
I don't think we need to include Alpha in the title -- the callout below is probably sufficient